microsoft-edge/web-platform/release-notes/133.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Microsoft Edge 133 web platform release notes (Feb. 2025)
+description: Microsoft Edge 133 web platform release notes (Feb. 2025).
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 02/06/2025
+---
+# Microsoft Edge 133 web platform release notes (Feb. 2025)
+
+The following are the new web platform features and updates in Microsoft Edge 133, which was released on February 6, 2025.
+
+To stay up-to-date and get the latest web platform features, download an Insiders preview version<!-- channel --> of Microsoft Edge; see [Become a Microsoft Edge Insider](https://aka.ms/microsoftedge).
+
+
+<!-- ====================================================================== -->
+## Edge DevTools
+
+For Edge DevTools, see [What's New in Microsoft Edge DevTools](../../devtools-guide-chromium/whats-new/whats-new.md).
+
+
+<!-- ====================================================================== -->
+## WebView2
+
+For WebView2, see [Release Notes for the WebView2 SDK](../../webview2/release-notes/index.md).
+
+
+<!-- ====================================================================== -->
+## Web platform features
+
+
+<!-- ------------------------------ -->
+#### Enabled features
+
+
+<!-- ---------- -->
+###### `Animation.overallProgress`
+
+Adds an `overallProgress` property to the [Animation](https://developer.mozilla.org/docs/Web/API/Animation) JavaScript class.
+
+This property provides a convenient and consistent representation of how far along an animation has advanced across its iterations and regardless of the nature of its [timeline](https://developer.mozilla.org/docs/Web/API/AnimationTimeline).
+
+
+<!-- ---------- -->
+###### `Atomics.pause`
+
+Adds the `Atomics.pause` method, to hint to the CPU that the current code is executing a spinlock.
+
+
+<!-- ---------- -->
+###### CSP hash reporting for scripts
+
+Complex web applications often need to keep a record of the subresources that they download, for security purposes.  In particular, upcoming industry standards and best practices (such as PCI-DSS v4)<!-- todo: link? --> require that web applications keep an inventory of all the scripts they download and execute.
+
+This feature builds on CSP<!-- todo: expand --> and the Reporting API to report the URLs and hashes (for CORS/same-origin)<!-- todo: punc. --> of all the script resources that the document loads.
```

Comment:
consider spelling out "CORS/same-origin)"

https://learn.microsoft.com/en-us/style-guide/punctuation/slashes - "Don't use a slash as a substitute for or, like product/service. If the UI uses a slash in this way, follow the interface in describing the label, but use or to describe the action in text."